src/lib/utils/meta_table_manager.cpp

```diff
@@ -87,37 +153,79 @@ std::shared_ptr<Table> MetaTableManager::generate_chunks_table() {
   return output_table;
 }
 
-std::shared_ptr<Table> MetaTableManager::generate_segments_table() {
+std::shared_ptr<Table> MetaTableManager::generate_chunk_sort_orders_table() {
   const auto columns = TableColumnDefinitions{{"table_name", DataType::String, false},
                                               {"chunk_id", DataType::Int, false},
                                               {"column_id", DataType::Int, false},
-                                              {"column_name", DataType::String, false},
-                                              {"column_data_type", DataType::String, false},
-                                              {"encoding_name", DataType::String, true}};
-  // Vector compression is not yet included because #1286 makes it a pain to map it to a string.
+                                              {"order_mode", DataType::String, false}};
   auto output_table = std::make_shared<Table>(columns, TableType::Data, std::nullopt, UseMvcc::Yes);
 
   for (const auto& [table_name, table] : Hyrise::get().storage_manager.tables()) {
     for (auto chunk_id = ChunkID{0}; chunk_id < table->chunk_count(); ++chunk_id) {
-      for (auto column_id = ColumnID{0}; column_id < table->column_count(); ++column_id) {
-        const auto& chunk = table->get_chunk(chunk_id);
-        const auto& segment = chunk->get_segment(column_id);
+      const auto& chunk = table->get_chunk(chunk_id);
+      const auto ordered_by = chunk->ordered_by();
+      if (ordered_by) {
+        std::stringstream order_by_mode_steam;
+        order_by_mode_steam << ordered_by->second;
+        output_table->append({pmr_string{table_name}, static_cast<int32_t>(chunk_id), static_cast<int32_t>(ordered_by->first),
+                            pmr_string{order_by_mode_steam.str()}});
+      }
+    }
+  }
 
-        const auto data_type = pmr_string{data_type_to_string.left.at(table->column_data_type(column_id))};
-        AllTypeVariant encoding = NULL_VALUE;
-        if (const auto& encoded_segment = std::dynamic_pointer_cast<BaseEncodedSegment>(segment)) {
-          encoding = pmr_string{encoding_type_to_string.left.at(encoded_segment->encoding_type())};
-        }
+  return output_table;
+}
+                                              {"sorted_by_column_name", DataType::String, false},
+                                              {"sort_order_mode", DataType::String, false}};
+  auto output_table = std::make_shared<Table>(columns, TableType::Data, std::nullopt, UseMvcc::Yes);
```

Comment:
This format seems completely broken.

Reply:
I accidentally committed half of a WIP function.

Reply:
Should be fixed.